notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6320 @@
+{
```

Comment:
<p>"...Elastic Cloud."</p><p> </p><p>"Ensure you have..."</p><p> </p><p>Maybe instead of referring users to the tweet, you can take a screenshot of it and include it here as an image so its less disruptive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>👍 PR to add the image <a href="https://huggingface.co/datasets/huggingface/cookbook-images/discussions/20" target="_blank">https://huggingface.co/datasets/huggingface/cookbook-images/discussions/20</a></p><p> </p>

Reply:
<p>The other two suggestions haven't been resolved yet</p>

Reply:
<p>👍</p>